tools/ci-scripts/check-aws-sdk-standalone-integration-tests

```diff
@@ -39,4 +39,10 @@ find "${tmp_dir}"
 
 pushd "${tmp_dir}/aws/sdk/integration-tests"
 cargo check --tests
+
+# Running WebAssembly (WASI) specific integration tests
+pushd "${tmp_dir}/aws/sdk/integration-tests/webassembly" &>/dev/null
+cargo wasi build
+env_credentials="--env AWS_ACCESS_KEY_ID='' --env AWS_SECRET_ACCESS_KEY='' --env AWS_SESSION_TOKEN=''"
```

Comment:
A mechanism to inject the AWS temporary credentials through the default environment variable is required in order to run the WebAssembly integration tests.

Reply:
This will not be necessary anymore as we are exercising an S3 API that does not need authentication.